README.md

```diff
@@ -191,13 +191,14 @@ Currently, the following clients are available for use:
   - ✔ Opus formats.
   - ❌ No mix/playlist/search support.
 - `ANDROID`
-  - ❌ Heavily restricted, frequently dysfunctional.
+  - ✔ Opus formats.
```

Comment:
I would prefer a disclaimer is left here so that users know of the risks of using this client, even if it does appear to work for now with these changes.